.env

```diff
@@ -1,4 +1,5 @@
 PORT=7080
-RAZZLE_BASE_PATH=http://localhost:7080
```

Comment:
When renaming this to `BASE_URL` we will also need to change it in the [infrastructure code](https://github.com/bbc/simorgh-infrastructure/blob/latest/Makefile#L7) aswell

Reply:
Have raised https://github.com/bbc/simorgh-infrastructure/issues/178